docs/styleguide.config.js

```diff
@@ -52,6 +52,12 @@ module.exports = {
     }
   ],
   components: '../react/**/*.jsx',
+  template: 'templateLato.html',
```

Comment:
I would just name it `template.html`.

Reply:
Sure, I hesitated on this more than i would admit.